### PR TITLE
skip refetching routes on window focus change

### PR DESCRIPTION
**Pull request by mintdart**

### CODE REVIEW

File: src/queries/useGetRoutes.tsx [hunk 1]
```diff
@@ -87,6 +87,8 @@ export function useGetRoutes({ chain, from, to, amount, extra = {} }: IGetListRo
 					],
 					queryFn: () => getAdapterRoutes({ adapter, chain, from, to, amount, extra }),
 					refetchInterval: 15_000,
+					refetchOnWindowFocus: false,
```
Comment (0xngmi) [inline src/queries/useGetRoutes.tsx:90]: why not? too many triggers?

**Review by vrtnd — APPROVED** [review]

File: src/queries/useGetRoutes.tsx (hunk 1 above)
Comment (mintdart) [reply src/queries/useGetRoutes.tsx:90]: yeah when user selects a route-> approves token, and then sometimes quote gets refreshed right before a swap

Comment (ai-slave) [reply src/queries/useGetRoutes.tsx:90]: If too many requests isn't a problem server side I'd still show a tip that "there are better routes now, wanna update?"

Comment (mintdart) [reply src/queries/useGetRoutes.tsx:90]: too many requests is a problem, sometimes aggregators rate limit and valid/better routes disappear after user approved a token

**Pull request merged**